packages/node_modules/@ciscospark/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -78,9 +78,9 @@ function makeMockMeeting(params) {
 
 describe(`plugin-calendar`, () => {
   describe(`#list()`, () => {
-    let createdMeeting, creator, mccoy, meetingParams, spark, spock;
+    let createdMeeting, creator, mccoy, meetingParams, mercuryPromise, spark, spock;
 
-    before(`create test users`, () => testUsers.create({
+    before(`create test users and initialize spark`, () => testUsers.create({
```

Comment:
I would break this into multiple named before blocks. that makes it a lot easier to debug issues in jenkins.